### ISSUE TITLE
Reproduce config

### ISSUE THREAD

**Issue by khoiucd** [MCG-NJU/FSL-Video#3]
Nice work!

Could you please release the training configs or instructions to reproduce the results (for all datasets) reported in the paper?

Thank you very much.

**Comment by zhuzhenxi** [comment]
For the pre-training of Baseline and Baseline Plus, most of our parameters are the same as the default parameters of TAM (wd=1e-4, batchsize 8x4, dropout 0.5). We set lr=1e-3 for all methods without using ImageNet pre-trained weights. When using ImageNet pre-trained weights, we set lr=1e-4 for classifier-based methods and lr=1e-5 for meta-learning methods.
